entities/differential.go

```diff
@@ -70,6 +74,18 @@ type DifferentialHunk struct {
 	Corpus    string `json:"corpus"`
 }
 
+// A DifferentialProperties contains dynamic metadata about the Diff.
+type DifferentialProperties struct {
+	ArcOnto []DifferentialArcOnto `json:"arc:onto"`
+}
+
+// A DifferentialArcOnto contains a list of branch destinations set with arc.
```

Comment:
nit: should this say 'contains a branch destination set with arc' rather than 'a list of...'?